src/scheduler/pbs_sched.c

```diff
@@ -1045,13 +998,28 @@ main(int argc, char *argv[])
 				break;
 		}
 	}
+	/* Check whether the sched name is specified in command line args
+	 * If specified then copy the sched name
+	 * */
+	if (argc > optind)
+		strncpy(sc_name, argv[optind], PBS_MAXSCHEDNAME - 1);
+	else {
+		strncpy(sc_name, PBS_DFLT_SCHED_NAME, PBS_MAXSCHEDNAME -1);
+		dflt_sched = 1;
+	}
```

Comment:
At present we can also run the scheduler in either of the following ways
Example:
    pbs_sched sc1 -d /var/spool/pbs    
    pbs_sched -d /var/spool/pbs sc1

where sc1 is name of the scheduler. If new option required still, I can go ahead and do this.

Reply:
No other pbs daemon takes an argument like this.  Setting the scheduler name should be a getopt() option.

Reply:
I have added new option to give the name of scheduler which "-I". one can start the scheduler as follows going forward
pbs_sched -I sc1 -S 15050

Reply:
This is fine, but I'm just curious.  Why -l instead of -n?

Reply:
Since capital letter 'S' is used for port, I initially thought of using capital letter 'N' but this is already reserved and hence used 'I' (stands for scheduler ID).  Please let me know if I still need to use '-n'.

Reply:
My only worry is that we call it a name everywhere else.  If we call it an id here, it could be confusing.  I'll leave it up to you to decide what is best.